client/dist/index.css

```diff
@@ -0,0 +1,62 @@
+*, *:before, *:after {
+  box-sizing: border-box;
+}
```

Comment:
I think because you had a working solution for the case where we did not use this, I may have broken some of your other styles